judge/management/commands/create_problem.py

```diff
@@ -12,12 +13,28 @@ def add_arguments(self, parser):
         parser.add_argument('body', help='problem description')
         parser.add_argument('type', help='problem type')
         parser.add_argument('group', help='problem group')
+        parser.add_argument('time_limit', help='time limit in (fractional) seconds')
+        parser.add_argument('memory_limit', help='memory limit in kilobytes')
+        parser.add_argument('points', help='problem points')
+        parser.add_argument('--partial-points', help='allow partial points',
+            action='store_true', default=False, dest='pr_pts')
+        parser.add_argument('--all-languages', help='allow all languages',
+            action='store_true', default=False, dest='all_lang')
 
     def handle(self, *args, **options):
         problem = Problem()
         problem.code = options['code']
         problem.name = options['name']
         problem.description = options['body']
         problem.group = ProblemGroup.objects.get(name=options['group'])
-        problem.types = [ProblemType.objects.get(name=options['type'])]
+        problem.time_limit = options["time_limit"]
+        problem.memory_limit = options["memory_limit"]
+        problem.points = options["points"]
+        if options['pr_pts']:
```

Comment:
Is this `if` necessary? It seems we can just do
```py
problem.partial = options['pr_pts']
```

Reply:
I wrote this for not assuming the default value of this field, i.e., if `Allows partial points` is enabled by default at some point, the code should still work, though I don't have strong opinions about this.